internal/engine/experiment/torsf/torsf.go

```diff
@@ -31,6 +43,15 @@ type TestKeys struct {
 
 	// Failure contains the failure string or nil.
 	Failure *string `json:"failure"`
+
+	// PersistentDatadir indicates whether we're using a persistent tor datadir.
+	PersistentDatadir bool `json:"persistent_datadir"`
+
+	// RendezvousMethod contains the method used to perform the rendezvous.
+	RendezvousMethod string `json:"rendezvous_method"`
+
+	// TorLogs contains the bootstrap logs.
+	TorLogs []string `json:"tor_logs"`
```

Comment:
We need to update the related ooni/spec to account for these new fields in the results.

Reply:
done! ✔️